spec/suites/layer/vector/PathSpec.js

```diff
@@ -106,4 +106,39 @@ describe('Path', function () {
 			expect(path._clickTolerance()).to.equal(0);
 		});
 	});
+
+	describe('#accessibility', function () {
+		it('should have title', function () {
+			var path = L.circleMarker([1, 2]);
+			path.addTo(map);
+
+			var element = path.getElement();
+
+			expect(element.querySelector('title').innerHTML).to.eql('CircleMarker');
```

Comment:
This would also need to change if the default changes.
```suggestion
			expect(element.querySelector('title').innerHTML).to.eql('Circle marker');
```

I love this! Hope it gets merged.